packages/server/src/routes/tenants.js

```diff
@@ -17,6 +17,40 @@ router.get('/', requireUser, async (req, res) => {
     res.json(response);
 });
 
+router.post('/', async (req, res) => {
```

Comment:
This route doesn't seem to check any kind of permission. I think we might want a new helper like `requireUSDRAdminUser` that calls `requireAdminUser` and additionally verifies the user is part of a hardcoded agency ID (`1`, I think).